server/src/infra/entities/person.entity.ts

```diff
@@ -36,6 +37,12 @@ export class PersonEntity {
   @Column({ default: '' })
   thumbnailPath!: string;
 
+  @Column({ type: 'uuid', nullable: true })
+  faceAssetId!: string | null;
+
+  @ManyToOne(() => AssetEntity, { onDelete: 'SET NULL', nullable: true })
+  faceAsset!: AssetEntity | null;
+
```

Comment:
I don't think you need nullable in both column decorators. Also, I don't think you need `type: 'uuid'` in the ID column either. Can you compare this with other relations and make it match?

Reply:
Nullable is set in the album entity:

```
@ManyToOne(() => AssetEntity, { nullable: true, onDelete: 'SET NULL', onUpdate: 'CASCADE' })
albumThumbnailAsset!: AssetEntity | null;

@Column({ comment: 'Asset ID to be used as thumbnail', nullable: true })
albumThumbnailAssetId!: string | null;
```

I set the `uuid` type because the server complained about something (sorry, I didn't keep the logs), but that doesn't seem to happen anymore.

Reply:
Sounds good. Looks like that is the only other nullable relation we have. 

Right, typeorm would have defaulted to varchar before you added the relation, which obviously would conflict with the foreign key constraint.